src/legacy/core_plugins/kibana_react/public/top_nav_menu/top_nav_menu.tsx

```diff
@@ -23,7 +23,7 @@ import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { I18nProvider } from '@kbn/i18n/react';
 import { TopNavMenuData } from './top_nav_menu_data';
 import { TopNavMenuItem } from './top_nav_menu_item';
-import { SearchBar, SearchBarProps } from '../search_bar';
+import { SearchBar, SearchBarProps } from '../../../../core_plugins/data/public';
```

Comment:
Should we expose the UI component via the search service, or is the best practice to statically import like this now? Asking mainly so I know what to do in our saved query PR.